cli/internal/kubernetes/upgrade.go

```diff
@@ -338,7 +339,7 @@ func (u *Upgrader) UpdateAttestationConfig(ctx context.Context, newAttestConfig
 // GetClusterAttestationConfig fetches the join-config configmap from the cluster, extracts the config
 // and returns both the full configmap and the attestation config.
 func (u *Upgrader) GetClusterAttestationConfig(ctx context.Context, variant variant.Variant) (config.AttestationCfg, *corev1.ConfigMap, error) {
```

Comment:
With my previous suggestion, this method can simply return `config.AttestationCfg`